tests/test_utils.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from bonfire.utils import (
+    split_equals,
+    validate_time_string,
+    get_version,
+    hms_to_seconds
+)
+
+
+@pytest.mark.parametrize(
+    "list_of_str, expected",
+    [
+        (
+            ["t1=test1", "t2=test2"],
+            {"t1": "test1", "t2": "test2"}
+        ),
+        (
+            ["company=redhat", "team=Dev Prod"],
+            {"company": "redhat", "team": "Dev Prod"}
+        ),
+    ]
+)
+def test_split_equals_pass(list_of_str: str, expected: str):
+    result = split_equals(list_of_str)
+
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    "list_of_str",
+    [
+        (
+            ["t1 = test1", "t2 = test2"],
+        ),
+        (
+            ["t1 test1", "t2 test2"],
+        ),
+    ]
+)
+def test_split_equals_fail(list_of_str: list):
+    with pytest.raises(Exception):
+        split_equals(list_of_str)
+
+
+@pytest.mark.parametrize(
+    "time, expected",
+    [
+        (
+            "30m", "30m"
+        ),
+        (
+            "1h45m", "1h45m"
+        ),
+        (
+            "1h20s", "1h20s"
+        ),
+        (
+            "2h15m32s", "2h15m32s"
+        ),
+    ]
+)
+def test_validate_time_string(time: str, expected: str):
```

Comment:
Could we add a test to assert that the validate FAILS and raises a ValueError for the 3 invalid cases written in that func?
* `"invalid format for duration '{time}', expecting h/m/s string. Ex: '1h30m'"
* `"invalid duration '{time}', must be less than 14 days"`
* `"invalid duration '{time}', must be more than 30 mins"`